convertToSingleHost.js

```diff
@@ -80,10 +80,16 @@ async function updatePackageJsonForSingleHost(host) {
 
     // remove scripts that are unrelated to the selected host
     Object.keys(content.scripts).forEach(function (key) {
-        if (key === "convert-to-single-host"
-        ) {
+        if (key === "convert-to-single-host") {
             delete content.scripts[key];
         }
+        if (key === "sideload" && host === "outlook") {
+            delete content.scripts[key];
+        }
+
+        if (key === "start" && host === "outlook") {
+            content.scripts[key] = "npm run build:dev && concurrently \"npm run start:server\"";
```

Comment:
It would be good to have a work item to update office-addin-debugging to support Outlook (skipping sideload which it doesn't support). This would be easy to do. This is fine for now, but it would be better to have the start script work across hosts instead of special casing in the templates.

This change is only providing support in the SSO template, but really this belongs in all of the templates.

Reply:
Opened work item: Task 3905711: Update office-addin-debugging to support Outlook